### ISSUE TITLE
New feature: Change default repo link in upper right corner

### ISSUE THREAD

**Issue by rydmike** [invertase/docs.page#387]
Is there a way to change the repo link shown in the upper right corner?
If not i would suggest a feature that enables this.

It by default links to the repo the docs are in, which is not very useful for my use case. 

For update and access convenience, I have chosen to not have the docs.page repo in the same repo as the package and app the documentation documents.

It would therefore be much more useful if it could be defined to be to another repo, so I can point it to the actual repo the docs document. That would be one more anchor link I can remove. Earlier I had it as an anchor link, but those seems broken, as posted in other issue.

![Screenshot 2024-10-06 at 17 31 28](https://github.com/user-attachments/assets/1ec3281b-4ce3-463b-9254-ccf5cf2b1bb7)

**Comment by Ehesp** [comment]
Yep that's a fair point, should be pretty trivial to add, maybe an optional githubCardRepo: 'org/repo' in the header config?

**Comment by rydmike** [comment]
Since Anchors are back again 😄 I'm as before, using it as place for a link to the repo the doc is about. 

But as mentioned, it would be nice to be able to remove it and have it in the intended place.

![Screenshot 2024-10-09 at 15 59 19](https://github.com/user-attachments/assets/f7a6afd7-75ef-4a5f-83c0-c26e9876b86f)
